### DISCUSSION TITLE
Investigate the addition of Chatterbox (E2EE website live chat)

### DISCUSSION THREAD

**Discussion by nourkagha** [chatwoot/chatwoot#5009] [Feature Requests]
**Chatterbox** is a newly launched, embedded and end-to-end encrypted live chat solution, based on Matrix.

All information here: https://element.io/blog/element-launches-chatterbox

**Comment by julianpistorius** [comment]
@nourkagha Do you (or anybody else here) have any ideas on how to integrate Chatterbox with Chatwoot?

Could one start with a vanilla Chatwoot installation and then add Chatterbox integration later?

Do you envision adding Chatterbox/Matrix as a [Chatwoot channel](https://www.chatwoot.com/docs/product/#channels)?

Background: I know very little about these two projects. I'm a maintainer of an open source project and we're looking at adopting Chatwoot or Chatterbox for user support. We already use Matrix for developer conversations and some user support. I'm intrigued by the possibility of combining the strengths of Chatwoot with Matrix, but I don't have enough background on their architectures to know if this is feasible at all.

Details: https://gitlab.com/exosphere/exosphere/-/issues/486

Thanks in advance for any guidance.
